.github/workflows/phpunit.yml

```
@@ -0,0 +1,53 @@
+name: phpunit tests
+
+on:
+  push:
```

It might be worth making this into a reusable workflow called by the Workflow PR to allow the PR workflow to fail fast if unit tests fail

---

Thank you, that is the ultimate plan :). note this PR is Draft. ;)